plugins/factory-plugin/src/theia-commands.ts

```diff
@@ -68,13 +69,13 @@ export class TheiaCloneCommand {
         this.projectsRoot = projectsRoot;
     }
 
-    execute(): PromiseLike<void> {
```

Comment:
i don't see any good reasons to rename `execute` to `clone`.
This is a command. At the moment we tell Che-theia to clone (running clone commands) at start up but we may chain other commands that are not about cloning.